### PR TITLE
add support of restPrefix and prefix for blueprint

### PR DESCRIPTION
**Pull request by alexpwf**
#8 fix proposal

### CODE REVIEW

File: test/generators.test.js [hunk 1]
```diff
@@ -335,6 +335,7 @@ describe('Generators', function () {
         var tags = generators.tags(models);
         var generatedRoutes = generators.routes(controllers, {routes: {'get /user/phone/:phoneNumber': 'UserController.phone'}}, tags);
         var actual = generators.paths(generatedRoutes, tags, {list: [{$ref: '#/parameters/PerPageQueryParam'}]});
+        console.log ('>>',generatedRoutes);
```
Comment (theo4u) [inline test/generators.test.js]: no need for this

Comment (alexpwf) [reply test/generators.test.js]: true

Comment (theo4u) [reply test/generators.test.js]: you can add a nice test case and pass `config.blueprint` props as the configuration for generator.routes function here (check the way `routes` object was passed in for `routes.js`)
 https://github.com/theo4u/sails-hook-swagger-generator/blob/a9d9862ba3db79076424749721fe4390ee672c22/test/generators.test.js#L163

**Comment by theo4u** [comment]
you can add a nice test case and pass `config.blueprint` props as the configuration for `generator.routes` function here (check the way `routes` object was passed in for `routes.js`), so you can add `{blueprints:{}}` for `blueprints.js
 https://github.com/theo4u/sails-hook-swagger-generator/blob/a9d9862ba3db79076424749721fe4390ee672c22/test/generators.test.js#L163

**Comment by alexpwf** [comment]
Take a look at this test, you'll tell me new ones.

**Comment by theo4u** [comment]
LGTM

**Pull request merged**